lang/src/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -14,41 +14,50 @@
 
 package org.partiql.lang
 
-import com.amazon.ion.*
-import org.partiql.lang.ast.*
-import org.partiql.lang.eval.*
-import org.partiql.lang.eval.builtins.*
+import com.amazon.ion.IonSystem
+import org.partiql.lang.ast.ExprNode
+import org.partiql.lang.ast.toAstStatement
+import org.partiql.lang.ast.toExprNode
+import org.partiql.lang.eval.Bindings
+import org.partiql.lang.eval.CompileOptions
+import org.partiql.lang.eval.EvaluatingCompiler
+import org.partiql.lang.eval.ExprFunction
+import org.partiql.lang.eval.ExprValueFactory
+import org.partiql.lang.eval.Expression
+import org.partiql.lang.eval.ThunkReturnTypeAssertions
+import org.partiql.lang.eval.builtins.createBuiltinFunctions
 import org.partiql.lang.eval.builtins.storedprocedure.StoredProcedure
 import org.partiql.lang.eval.visitors.PipelinedVisitorTransform
 import org.partiql.lang.eval.visitors.StaticTypeInferenceVisitorTransform
 import org.partiql.lang.eval.visitors.StaticTypeVisitorTransform
-import org.partiql.lang.syntax.*
-import org.partiql.lang.util.interruptibleFold
-import org.partiql.lang.types.StaticType
+import org.partiql.lang.syntax.Parser
+import org.partiql.lang.syntax.SqlParser
 import org.partiql.lang.types.CustomType
+import org.partiql.lang.types.StaticType
+import org.partiql.lang.util.interruptibleFold
 
 /**
  * Contains all of the information needed for processing steps.
  */
 data class StepContext(
-    /** The instance of [ExprValueFactory] that is used by the pipeline. */
-    val valueFactory: ExprValueFactory,
+        /** The instance of [ExprValueFactory] that is used by the pipeline. */
+        val valueFactory: ExprValueFactory,
 
-    /** The compilation options. */
-    val compileOptions: CompileOptions,
+        /** The compilation options. */
+        val compileOptions: CompileOptions,
 
-    /**
-     * Returns a list of all functions which are available for execution.
-     * Includes built-in functions as well as custom functions added while the [CompilerPipeline]
-     * was being built.
-     */
-    val functions: @JvmSuppressWildcards Map<String, ExprFunction>,
+        /**
+         * * Returns a list of all functions which are available for execution.
```

Comment:
(This might get fixed in https://github.com/partiql/partiql-lang-kotlin/issues/43 when we add ktlint)

Not sure why additional whitespace is being added here and on L132 and L295-296

Reply:
Thanks for pointing these out; neither am I. Fixed!